diff --git a/node_modules/@capacitor-mlkit/barcode-scanning/CapacitorMlkitBarcodeScanning.podspec b/node_modules/@capacitor-mlkit/barcode-scanning/CapacitorMlkitBarcodeScanning.podspec
index 3710d96..32224f3 100644
--- a/node_modules/@capacitor-mlkit/barcode-scanning/CapacitorMlkitBarcodeScanning.podspec
+++ b/node_modules/@capacitor-mlkit/barcode-scanning/CapacitorMlkitBarcodeScanning.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'GoogleMLKit/BarcodeScanning', '5.0.0'
+  s.dependency 'GoogleMLKit/BarcodeScanning', '6.0.0'
   s.swift_version = '5.1'
   s.static_framework = true
 end
